### PR TITLE
ui & api. separate admin users to standard users

### PR DESCRIPTION
**Pull request by federicoballarini**
Separate admin users and standard users counters into WebTop 5 dashboard.
Refer to https://github.com/NethServer/dev/issues/6051

### CODE REVIEW

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -64,7 +64,8 @@
         "webtop_tasks": "Tasks",
         "webtop_admin_password": "WebTop admin password",
         "warning": "Warning",
-        "change_default_password": "The default password for the admin user is 'admin'. Please change it as soon as possible from WebTop application to avoid security issues!"
+        "change_default_password": "The default password for the admin user is 'admin'. Please change it as soon as possible from WebTop application to avoid security issues!",
+        "active_admins": "Active admins"
```
Comment (edospadoni) [inline ui/public/i18n/language.json]: I think we can write only `Admins` here, without `Active`

**Comment by federicoballarini** [comment]
Maybe we can change the query as follow:
`select user_id from core.users where user_id = 'admin' and enabled = 't'`
A user can understand if an admin is disabled in webtop admin panel.

Same for users:
`select user_id from core.users where type = 'U' and user_id != 'admin' and enabled = 't'`

**Comment by lucagasparini** [comment]
> The code seems good, thank you!
> 
> I was wondering if showing the number of admins could be misleading for the user.
> All installations will always have the number of admins set to `2`. What is the value of showing such number inside the UI?
> 
> What you think about it @lucagasparini ?

I agree that showing the number of active admins does NOT provide any useful information and can only create misunderstandings.
In my opinion it can be completely removed from the interface.

**Comment by federicoballarini** [comment]
Ok, do you think we have to count NethServer admin into users count?
And what about flag `enabled = 't'`?

**Comment by lucagasparini** [comment]
> Ok, do you think we have to count NethServer admin into users count?
> And what about flag `enabled = 't'`?

I would say no.
I think the count of active users is as good as it has already been done by excluding admins users.

**Comment by federicoballarini** [comment]
I'm going to remove admin counter.

**Pull request merged**